src/index.html

```diff
@@ -9,6 +9,26 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="box">
+        <a href="#">
+          <img
+           src="images/logo.png"
+           alt="logo"
+           class="logo"
+          >
```

Comment:
```suggestion
          <img
            src="images/logo.png"
            alt="logo"
            class="logo"
          >
```
2 spaces indentation